# Throws
# error: implicit declaration of function 'snprintf' [-Wimplicit-function-declaration]
# on the RISCV builder.
--- a/src/formats/svg.c
--- b/src/formats/svg.c
@@ -6,6 +6,7 @@
 
 #include <ctype.h>
 #include <string.h>
+#include <stdio.h>
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wexpansion-to-defined"
